web/static/js/dribbble.js

```diff
@@ -17,24 +17,25 @@ function composeDribbbleShot(originalElement, url, imageUrl) {
 }
 
 $(function() {
-  const listSelector = "[data-role=dribbble-list]";
-  const shotSelector = "[data-role=dribbble-shot]";
-  const list = $(listSelector);
-  const basicElement = $(shotSelector);
-  const accessToken = list.data("access-token");
-  const bucketId = list.data("bucket-id");
-  const count = list.data("count");
+  const $list = $("[data-role=dribbble-list]");
+  const accessToken = $list.data("accessToken");
 
-  $.ajax({
-    url: dribbleBucketUrl({bucket: bucketId, token: accessToken, count: count}),
-    context: document.body
-  }).done(function(data) {
-    $.each(data, function(index, shot) {
-      console.log(shot);
-      let shotElement = composeDribbbleShot(basicElement, shot.html_url, shot.images.hidpi);
-      shotElement.show();
-      list.append(shotElement);
+  if(accessToken) {
```

Comment:
These changes look pretty unrelated to the organization. Should this be a different PR?

Reply:
Yep, I'll put it in another PR. Thanks.